xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEFilter.cpp

```diff
@@ -192,7 +192,7 @@ int CActiveAEFilter::ProcessFilter(uint8_t **dst_buffer, int dst_samples, uint8_
     frame->format = m_sampleFormat;
 
     result = avcodec_fill_audio_frame(frame, channels, m_sampleFormat,
-                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 0);
+                             src_buffer[0], channels * src_samples * av_get_bytes_per_sample(m_sampleFormat), 1);
```

Comment:
@fernetMenta: Nope they don't have as the m_frames of pulseaudio might be something like: 2205 and then there is a problem. So the m_frames directly influences the src_samples. In our case 2205 * 8 = 17640 which is not dividable by 32. So fill frame fails cause it wants 17664. We cannot assume the m_frames coming from a Sink is mod 32.